index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="shortcut icon" href="images/favicon_1.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,900&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="logo">
+                My Logo
+            </div>
+            <div class="menu">
+                <img src="images/Normal Button/Tertiary/Icons/Union.png" alt="">
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="main_container">
+               <h1>I’m Taflon Glad to see you!</h1> 
+               <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+
+               </p>
+               <a href="#">
+                LET’S CONNECT
+               </a>
```

Comment:
- [ ] For semantic reasons, since this text here is not a link, You may consider using a div, span instead of an anchor tag